src/animationBuilder.tsx

```diff
@@ -25,12 +26,50 @@ const mockTargetValues: LayoutAnimationsValues = {
   currentBorderRadius: 0,
 };
 
+function getCommonProperties(
+  layoutStyle: StyleProps,
+  secondStyle: StyleProps | Array<StyleProps>
+) {
+  const secondStyleFlat = Array.isArray(secondStyle)
+    ? secondStyle.flat()
+    : [secondStyle];
+
+  let commonKeys: Array<string> = [];
+  secondStyleFlat.forEach((s) => {
+    if ('initial' in s) {
+      s = s.initial.value; // Include properties of animated style
+    }
+
+    const commonStyleKeys = Object.keys(s).filter((key) => key in layoutStyle);
+    commonKeys = commonKeys.concat(...commonStyleKeys);
```

Comment:
It should work without `...`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat
```suggestion
    commonKeys = commonKeys.concat(commonStyleKeys);
```